### PR TITLE
update CellCycleScoring Function

### PR DESCRIPTION
**Pull request by samuel-marsh**
Hi Seurat Team,

This PR is in reference to issue #3609.  Checked the function and appears to be working.  Wiki doesn't specify whether you prefer users to roxygenize changes prior to submitting PR or do it on your end so I just edited the code and description.  Can easily roxygenize on my end if you prefer.

Best,
Sam

### CODE REVIEW

**Comment by samuel-marsh** [comment]
Not sure why the checks failed but appears to be unrelated to the actual code in the PR?  Thoughts?

Best,
Sam

File: R/utilities.R [hunk 1]
```diff
@@ -445,16 +449,22 @@ CellCycleScoring <- function(
...
   set.ident = FALSE,
   ...
 ) {
   name <- 'Cell.Cycle'
   features <- list('S.Score' = s.features, 'G2M.Score' = g2m.features)
+  if(is.null(ctrl)) {
+    ctrl <- min(vapply(X = features, FUN = length, FUN.VALUE = numeric(length = 1)))
+  } else {
+    ctrl <- ctrl
+  }
```
Comment (mojaveazure) [inline R/utilities.R]: The `else` part of this if/else is not necessary. Also, adjust the style for proper spacing and naming of arguments.
```suggestion
  if (is.null(x = ctrl)) {
    ctrl <- min(vapply(X = features, FUN = length, FUN.VALUE = numeric(length = 1)))
  }
```

**Review by mojaveazure — CHANGES_REQUESTED** [review]
Hi Sam,

This looks great, thank you! I've made a suggestion below regarding style, but other than that I'm happy with the fix.

**Comment by samuel-marsh** [comment]
Sounds good! Just committed the suggestion.  Also do you prefer to run `roxygenize` on your end for the update to manual entry or should I run and add commit?

Thanks!
Sam

**Comment by mojaveazure** [comment]
I've taken care of updating the help pages. We import documentation from packages not automatically installed when installing Seurat. Rather than having you cherry pick which aspects of the documentation to commit, I've handled it just to ensure that no bits of the documentation get lost. This will change in the future as we move away from this fragile design.

**Review by mojaveazure — APPROVED** [review]

**Comment by mojaveazure** [comment]
Thanks for taking care of this, Sam!

**Comment by samuel-marsh** [comment]
No problem, happy to help!!

**Pull request merged**